pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml

```diff
@@ -22,6 +22,17 @@ data:
         tag journald
       </rule>
     </match>
+    <filter journald>
+      @type record_transformer
+      enable_ruby true
+      <record>
+       #k8s_apiserver_apiserver-vqxg4_kube-service-catalog_72ce4d73-5224-11e9-98d0-000d3a196756_0
+       CONTAINER ${unless record["CONTAINER_NAME"].nil? ; record["CONTAINER_NAME"].split("_")[1] ; else "" ; end; }
```

Comment:
Is this 1-indexed or is the [0] field special? I guess I don't actually know what the source data looks like.

Reply:
data format is comment field above. so first field [0] is `k8s_`

Reply:
comment is not very informative :/

Reply:
Ahh, makes sense. Maybe we should update the comment to be clearer? :)